src/everything/everything.ts

```diff
@@ -223,7 +202,7 @@ export const createServer = () => {
       if (!logsUpdateInterval) {
           console.error("Starting logs update interval");
           logsUpdateInterval = setInterval(async () => {
-          await server.sendLoggingMessage( messages[Math.floor(Math.random() * messages.length)], sessionId);
+          await server.sendLoggingMessage(messages[Math.floor(Math.random() * messages.length)]);
```

Comment:
```suggestion
          await server.sendLoggingMessage(messages[Math.floor(Math.random() * messages.length)], sessionId);
```

Why did you remove the `sessionId` argument from every call to `sendLoggingMessage`?

This is the mechanism that allows multiple clients to have different logging levels set and respected. 

You wouldn't have been able to test with the current PR, because the logging capability was removed, but after you put that back in, try opening multiple Inspector instances connected to the same Everything server. Only the latest requested level will be respected. Whereas if you put the `sessionId` params back in the `sendLoggingMessage` calls, you'll see that each instance can request its own level and messages above that level will not be sent to it.